### DISCUSSION TITLE
Using #if to print header section at the top of each page of a PDF

### DISCUSSION THREAD

**Discussion by Muirik** [handlebars-lang/handlebars.js#1787] [Q&A]
I need to reprint a header section of a document for each page that ends up being printed. 

In other words, depending on how much data a particular document has, it may end up being printed over two or three pages when printed as a PDF. So in that case I want to print the header pertaining to that document for each page that ends up being printed for that document. 

Is there a way to accomplish this in `handlebars.js`, perhaps using something `#if`? If not, is there a different way I could accomplish it?

In the `style` section of my handlebars.js document this makes sure that margin is added to each page that gets printed:

```
      <style>
        @page {
          margin: 10px 0; 
        }
        // ... other styles
      </style>
```

So, I was hoping I could do something like this:

```
      {{#if @page}}
        <div class="header">
          <h3>Session Header - {{id}}</h3>
        </div>  
      {{/if}}
```

But, while this doesn't produce any errors, the section in question doesn't get printed at all. Any ideas of what I can use to accomplish this?

**Comment by jaylinski** [comment]
Handlebars.js was primarily made to render HTML for the browser and is therefore not aware of pages, so I'm afraid this is not possible with this library.
